code/game/objects/effects/decals/Cleanable/humans.dm

```diff
@@ -20,6 +20,7 @@ var/global/list/image/splatter_cache=list()
 	var/basecolor="#A10808" // Color when wet.
 	var/list/datum/disease2/disease/virus2 = list()
 	var/amount = 5
+	appearance_flags = NO_CLIENT_COLOR
```

Comment:
I still find issue with the fact that this makes blood uncolorable for all screen color matrices, not just the noire glasses.